src/tests/ftest/rebuild/pool_destroy_rebuild.yaml

```diff
@@ -0,0 +1,63 @@
+# change host names to your reserved nodes, the
+# required quantity is indicated by the placeholders
+hosts:
+  test_servers:
+    - server-A
+    - server-B
```

Comment:
I think it needs more than 2 servers for the rebuild tests, specially rebuilding the rank-3 server.

Reply:
There are more than one server started on each node. In total there will be 4 server ranks as there is servers_per_host: 2

Reply:
Thanks for the explanation. All of the current rebuild tests are on single server per host, rebuild on multiple-servers on the same host might causing DAOS-5868. 
We might need to add more tests for just restart server on the same host without the condition (pool destroy).